canonical-websites/cn.ubuntu.com
# Integrate vanilla v1

## Done

- Swap out old theme and styling for latest Vanilla
- Clean out redundant JS files
- Tidy up indentation
- Convert header and footer
- Convert markup on homepage

## QA

- Pull code
- Sanity check file deletions
- Run `./run serve --watch`
- View homepage on http://0.0.0.0:8010/

Fixes https://github.com/ubuntudesign/vanilla-squad/issues/73,
Fixes https://github.com/ubuntudesign/vanilla-squad/issues/62
